roles/openshift_hosted_logging/handlers/main.yml

```diff
@@ -3,3 +3,24 @@
   systemd: name={{ openshift.common.service_type }}-master state=restarted
   when: (openshift.master.ha is not defined or not openshift.master.ha | bool) and (not (master_service_status_changed | default(false) | bool))
   notify: Verify API Server
+
+- name: Verify API Server
+  # Using curl here since the uri module requires python-httplib2 and
```

Comment:
It looks like python-httplib2 is no longer a requirement as of Ansible 2.1.

I don't see a way to set the ca cert for the uri module, but I don't see much of an issue with disabling cert validation for this case.